modules/virt-about-vtpm-devices.adoc

```diff
@@ -10,9 +10,30 @@ A virtual Trusted Platform Module (vTPM) device functions like a
 physical Trusted Platform Module (TPM) hardware chip.
 You can use a vTPM device with any operating system, but Windows 11 requires
 the presence of a TPM chip to install or boot. A vTPM device allows VMs created
-from a Windows 11 image to function without a physical TPM chip.
+from a Windows 11 image to function without a physical TPM chip. A vTPM device also protects virtual machines by storing secrets without physical hardware.
 
-A vTPM device also protects virtual machines by storing secrets without physical hardware. {VirtProductName} supports persisting vTPM device state by using Persistent Volume Claims (PVCs) for VMs. You must specify the storage class to be used by the PVC by setting the `vmStateStorageClass` attribute in the `HyperConverged` custom resource (CR):
+{VirtProductName} supports persisting vTPM device state by using Persistent Volume Claims (PVCs) for VMs. If you do not specify the storage class for this PVC, {VirtProductName} uses the default storage class for virtualization workloads. If the default storage class for virtualization workloads is not set, {VirtProductName} uses the default storage class for the cluster.
+
+[NOTE]
+====
+The storage class that is marked as default for virtualization workloads has the annotation `storageclass.kubevirt.io/is-default-virt-class` set to "true". You can find this storage class by running the following command:
+
+[source,terminal]
+----
+$ oc get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
+----
+
+Similarly, the default storage class for the cluster has the annotation `storageclass.kubernetes.io/is-default-class` set to "true". To find this storage class, run:
+
+[source,terminal]
+----
+$ oc get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
+----
+
+To ensure consistent behavior, configure only one storage class as the default for virtualization workloads and for the cluster respectively.
+====
+
+Because this behavior might change in the future, it is recommended that you specify the storage class explicitly by setting the `vmStateStorageClass` attribute in the `HyperConverged` custom resource (CR):
```

Comment:
This is a personal concern but I don't know if I would voice it in upstream documentation. I'd say it's still recommended to set `vmStateStorageClass` as it's the intended way to define the storage class for backend PVCs, but don't know if it's appropriate to say that the fallback behavior could change in the future.

Reply:
Thank you, that makes sense. I will rephrase it.